src/main.ts

```diff
@@ -58,13 +58,7 @@ async function run(): Promise<void> {
           graalVMHome = await graalvm.setUpGraalVMJDKCE(javaVersion)
           break
         case c.DISTRIBUTION_MANDREL:
-          if (graalVMVersion.startsWith(c.MANDREL_NAMESPACE)) {
-            graalVMHome = await setUpMandrel(graalVMVersion, javaVersion)
-          } else {
-            throw new Error(
-              `Mandrel requires the 'version' option (see https://github.com/graalvm/setup-graalvm/tree/main#options).`
```

Comment:
You probably still want to throw this error if `graalVMVersion.length == 0`, no? If I read this correctly, `version: 'mandrel-22.2.0.0-Final'` and `version: '22.2.0.0-Final'` will now do the same thing, but `version` is still required.